core/src/test/java/org/opensearch/sql/planner/physical/PhysicalPlanTestBase.java

```diff
@@ -20,23 +20,18 @@
 import org.opensearch.sql.data.model.ExprValueUtils;
 import org.opensearch.sql.data.type.ExprCoreType;
 import org.opensearch.sql.exception.ExpressionEvaluationException;
-import org.opensearch.sql.expression.DSL;
 import org.opensearch.sql.expression.Expression;
 import org.opensearch.sql.expression.ReferenceExpression;
-import org.opensearch.sql.expression.config.ExpressionConfig;
 import org.opensearch.sql.expression.env.Environment;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit.jupiter.SpringExtension;
 
 @Configuration
 @ExtendWith(SpringExtension.class)
-@ContextConfiguration(classes = {ExpressionConfig.class})
+@ContextConfiguration(classes = {PhysicalPlanTestBase.class})
```

Comment:
This could be safely removed. All tests pass with and without it, nothing skipped.
2453 tests with `./gradlew :core:test`. Jacoco is also 100%.

Reply:
Sure, let me try. Thanks!

Reply:
Fixed in https://github.com/opensearch-project/sql/pull/1085/commits/91d4d5b2c3c94ad1e0b3d0278b1053e9dc7c81b9.